openreview/venue_request/process/deployProcess.py

```diff
@@ -390,4 +390,18 @@ def process(client, note, invitation):
             signatures = ['~Super_User1']
         )
 
-        client.post_invitation(matching_invitation)
\ No newline at end of file
+        client.post_invitation(matching_invitation)
+
+    client.post_invitation(openreview.Invitation(
+        id=SUPPORT_GROUP + '/-/Request' + str(forum.number) + '/Comment',
+        super=SUPPORT_GROUP + '/-/Comment',
+        reply={
+            'forum': forum.id,
+            'referent': forum.id,
```

Comment:
here is the issue, @purujitgoyal this should be `replyto` instead. 

Could you fix it and add a test?

Reply:
`replyto` must be None in order the comment to be present in all the replies.

Reply:
what about `forum`?

Reply:
and test for adding a new PC and posting a comment?

Reply:
forum should stay, the invitation should only replies to `forum.id` only.

> and test for adding a new PC and posting a comment?

That would be great, I added a comment in the new PR, you can also tests the existing replies were correctly updated.